client/src/components/App.js

```diff
@@ -25,10 +25,18 @@ const App = (props) => {
     fetchCurrentUser()
   }, [])
 
+  let message = ""
+  if(currentUser){
+    message = `${currentUser.firstName}`
+  }
+  
   return (
     <Router>
       <TopBar user={currentUser} />
       <Switch>
+        <Route exact path="/">
+          <h2>Hello from react {message}</h2>
+        </Route>
```

Comment:
lets just delete any sort of generic message. Just delete all of these relevant lines.

If you would like, in a future feature you can render a `LandingPage` component at the root path. For now, just get rid of it all!